src/rust.rs

```diff
@@ -396,7 +396,6 @@ impl Runtime {
     pub fn evaluate_script(&self, glob: HandleObject, script: &str, filename: &str,
                            line_num: u32, rval: MutableHandleValue)
                     -> Result<(),()> {
-        let filename_cstr = ffi::CString::new(filename.as_bytes()).unwrap();
```

Comment:
Because this variable is removed, the `filename_cstr` is still used in line 403 so CI failed.

I think one more thing we need to do here is to use `filename` directly in line 403.

https://github.com/servo/rust-mozjs/blob/d88435c3664edb280eca2f6b2610285892545839/src/rust.rs#L402-L404

Reply:
Thanks a lot for the feedback @CYBAI